examples/src/main/java/ai/djl/examples/inference/sr/SuperResolution.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package ai.djl.examples.inference.sr;
+
+import ai.djl.Application;
+import ai.djl.ModelException;
+import ai.djl.engine.Engine;
+import ai.djl.inference.Predictor;
+import ai.djl.modality.cv.Image;
+import ai.djl.modality.cv.ImageFactory;
+import ai.djl.modality.cv.util.NDImageUtils;
+import ai.djl.ndarray.NDArray;
+import ai.djl.ndarray.NDArrays;
+import ai.djl.ndarray.NDList;
+import ai.djl.ndarray.NDManager;
+import ai.djl.ndarray.types.DataType;
+import ai.djl.repository.zoo.Criteria;
+import ai.djl.repository.zoo.ZooModel;
+import ai.djl.training.util.ProgressBar;
+import ai.djl.translate.TranslateException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public final class SuperResolution {
+
+    private static final Logger logger = LoggerFactory.getLogger(SuperResolution.class);
+
+    private SuperResolution() {}
+
+    public static void main(String[] args) throws ModelException, TranslateException, IOException {
+        String imagePath = "src/test/resources/";
+        ImageFactory imageFactory = ImageFactory.getInstance();
+
+        List<Image> inputImages =
+                Arrays.asList(
+                        imageFactory.fromFile(Paths.get(imagePath + "fox.png")),
```

Comment:
Can you just link to original fox and monkey image?

Reply:
I just added them to the repo temporarily. Just for the review. I modified the images however, so I think we should put them in S3.